aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -230,13 +230,9 @@ public void onResult(SignInResult result) {
                         )
                         : null;
 
-                if (result.getSignInState().equals(SignInState.DONE)) {
-                    state = AuthSignedInStatus.SIGNED_IN;
-                } else {
-                    state = AuthSignedInStatus.SIGNED_OUT;
-                }
-
-                onSuccess.accept(new AuthSignInResult(state, newDetails));
+                onSuccess.accept(
+                        new AuthSignInResult(AuthSignInState.getEnum(result.getSignInState().toString()), newDetails)
```

Comment:
Note that in the future, depending on the API naming meeting, I may need to write a map to convert Cognito sign in state values to match ours but for now, per the team's agreement, we're starting with the same values.